build/configs/esp32_DevKitC/tash_wlan/defconfig

```diff
@@ -198,16 +198,16 @@ CONFIG_ESPTOOLPY_AFTER_RESET=y
 CONFIG_ESPTOOLPY_AFTER="hard_reset"
 CONFIG_ESP32_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p0"
 CONFIG_ESP32_AUTOMOUNT_USERFS_MOUNTPOINT="/mnt"
+CONFIG_ARCH_USE_FLASH=y
 
 #
 # Board-Partition Options
 #
-CONFIG_ARCH_USE_FLASH=y
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="768,"
-CONFIG_FLASH_PART_TYPE="smartfs,"
-CONFIG_FLASH_PART_NAME="user,"
+CONFIG_FLASH_PART_SIZE="768,32,"
+CONFIG_FLASH_PART_TYPE="smartfs,config,"
+CONFIG_FLASH_PART_NAME="user,config"
```

Comment:
It's better to rename the partition name other than "config".

Reply:
rename it  to nvram as artik.